flutter_cache_manager/pubspec.yaml

```
@@ -17,7 +17,7 @@ dependencies:
   path: ^1.8.0
   path_provider: ^2.0.0
   pedantic: ^1.10.0
-  rxdart: ^0.26.0
+  rxdart: ^0.27.0
```

If there are no breaking changes for this package (doesn't look like it as this is the only change), I'd prefer to support both versions.
```suggestion
  rxdart: ">=0.26.0 <0.28.0"
```